rules_gapic/bazel/src/main/java/com/google/api/codegen/bazel/BazelBuildFileView.java

```diff
@@ -232,6 +232,7 @@ private String replaceLabelName(String labelPathAndName, String newLabelName) {
         goImports.add(replaceLabelName(protoImport, ":longrunning_go_gapic"));
```

Comment:
Probably remove or comment out with explanation why gapic does not work and what should be done to make it work.

Reply:
Good idea, opened an issue for it too. Done.